geckodriver.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: geckodriver
+  version: 0.35.0
+  epoch: 0
+  description: A tool for exploring each layer in a docker image
+  copyright:
+    - license: MPL-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - openssl-dev
+      - rust
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/mozilla/geckodriver
+      tag: v${{package.version}}
+      expected-commit: 53f2b320033668644eae58a8f3af5c94f0efc0f5
+
+  - name: Configure and Build
+
+  - runs: |
+      mkdir -p ${{targets.destdir}}/usr/bin/
+      cargo build --release
+      mv target/release/geckodriver ${{targets.destdir}}/usr/bin/
+
+update:
+  enabled: true
+  github:
+    identifier: mozilla/geckodriver
+    strip-prefix: v
+
+test:
+  pipeline:
+    # AUTOGENERATED
+    - runs: |
+        geckodriver --version
+        geckodriver --help
```

Comment:
Can we do a selenium test with geckodriver here?

Reply:
done